aspnetcore/signalr/javascript-client.md

```diff
@@ -261,9 +261,47 @@ The following code demonstrates a typical manual reconnection approach:
 1. A function (in this case, the `start` function) is created to start the connection.
 1. Call the `start` function in the connection's `onclose` event handler.
 
-[!code-javascript[](javascript-client/samples/3.x/SignalRChat/wwwroot/chat.js?range=30-40)]
+[!code-javascript[](javascript-client/samples/3.x/SignalRChat/wwwroot/chat.js?range=30-46)]
 
-A real-world implementation would use an exponential back-off or retry a specified number of times before giving up.
+A production implementation would use an exponential back-off or retry a specified number of times before giving up.
+
+### Browser sleeping tab
```

Comment:
```suggestion
## Browser sleeping tab
```

This was under the "Reconnect clients" section, not sure if it belongs under its own heading, but reconnect doesn't seem correct